### PR TITLE
Test new upstream build config

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

See https://gitlab.com/ljn917/rdchiral_cpp/-/issues/1#note_718833726 for context.

### CODE REVIEW

**Comment by hadim** [comment]
ping @ljn917

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

**Comment by ljn917** [comment]
Thanks a lot! Everything works now.

**Comment by hadim** [comment]
Yes. I can only merge here on a new git tag such as 1.1.2 (we should avoid 1.1.1-2).

**Comment by ljn917** [comment]
Let me change the tag.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://gitlab.com/ljn917/rdchiral_cpp/-/archive/{{ git_version }}/rdchiral_cpp-{{ git_version }}.tar.gz
-  sha256: 01fadcea35c3c592f315bf2c7855820065242820739b9756fc15567f999c1b51
+  sha256: bebb41b43aeabf2800bd4711068bd383b06a05b93b570bef2b3d8ed290f4c33f
 
 build:
```
Comment (hadim) [inline recipe/meta.yaml:13]: Whenever you release a new version, you can make a PR to this repo changing the above config (`git_version` and `sha256`).

**Comment by hadim** [comment]
Let's wait for the CI to be green and I'll merge here.

**Comment by hadim** [comment]
Thanks @ljn917 for the new build config and thanks @maclandrol for the OSX confirmation!

**Pull request merged**